src/components/Button/Button.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import styles from 'components/Button/Button.module.scss'
+
+export const Button = ({ HandleClick, buttonName = 'submit' }) => (
```

Comment:
`buttonName` Is unnecessary property. Instead of this you should use `children`. Cause inside of button we can have different contet.
Example: 
```
<Button> <Fragment><Icon /> Some text <Fragment/> <Button/>
```

Reply:
fixed it